stock_demand_estimate/models/date_range.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+
+
+class DateRange(models.Model):
+    _inherit = "date.range"
+
+    @api.multi
+    @api.depends('date_start', 'date_end')
+    def _compute_days(self):
+        for rec in self:
+            if rec.date_start and rec.date_end:
+                rec.days = (fields.Date.from_string(rec.date_start) -
+                            fields.Date.from_string(rec.date_end)).days + 1
+
```

Comment:
I wonder if it wouldn't be even better to get the absolute value of the difference:

`rec.days = abs((fields.Date.from_string(rec.date_end) - fields.Date.from_string(rec. date_start)).days) + 1`

Because, as far as I can see (but I could be wrong), there is nothing in the _date_range_ module that constraints _date_start_ to be earlier than _date_end_ .